composer.json

```diff
@@ -12,10 +12,10 @@
         "react/socket-client": "^0.5.0",
         "mkraemer/react-pcntl": "2.0.*",
         "monolog/monolog": "^1.3",
-        "paragonie/random_compat": "^2"
+        "paragonie/random_compat": "^2",
+        "guzzlehttp/psr7": "1.2.3"
```

Comment:
Sorry, but i don't understand if I've done something good or bad ;)
Should i remove `guzzlehttp`?
My goal was to receive same libraries and versions when installing only `php-pm` and our app which is using `php-pm`. If i remember correctly i needed this `guzzlehttp` entry.
If something needs to be changed pls just write how You see it :)

I also added `react/socket` with last commit.